File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ClassicDiningTable.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ClassicDiningTable : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a classic dining table");
        }
    }
}

File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ClassicDiningTable.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ClassicDiningTable implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a classic dining table\n";
    }
}